src/forms.py

```diff
@@ -20,14 +31,56 @@ class CreateForm(SanicForm):
     submit = SubmitField('Create')
 
 
-@forms_blueprint.route('/create', methods=['GET', 'POST'])
-async def create_link(request):
+class UpdateForm(SanicForm):
+    url = StringField('URL', validators=[DataRequired()])
+    submit = SubmitField('Update')
+
+
+@form_blueprint.route('/create', methods=['GET', 'POST'])
+@login_required
+async def create_link(request, user):
     form = CreateForm(request)
-    if request.method == 'POST' and form.validate():
-        endpoint = form.endpoint.data
-        url = form.url.data
-        print('New link: Endpoint = {}, URL = {}'.format(endpoint, url))
-        return redirect('/')
+    if (request.method == 'POST') and form.validate():
+        data = [(
+            str(uuid.uuid1())[:36],
```

Comment:
Identifiers distinguish links with identical endpoints by having a time based element, which is present in its first part.
The longer the identifier, the more annoying it will be for people working with JSON formatted data from the API.

Reply:
Well, I am not sure if I agree with you since people are probably just going to use the API programmatically (and then not care about the length) or just copy&paste the IDs... But this is not the question. I was more _curious_ why do you ask just for the first 36 characters *explicitly* when the UUID is already guaranteed to be 36 characters long. I mean, from my point of view, the code would be equivalent if you remove the brackets...

Reply:
I was paranoid 😄 
Changed